test/e2e/View.test.ts

```diff
@@ -141,6 +141,8 @@ describe('BPMN Visu JS', () => {
     expect(graph.graph.model.cells.hasOwnProperty('startEvent_1')).toBeTruthy();
```

Comment:
:warning: should be removed as we now do a `expectModelContainsCell` on this cell which do the same `except` already
```suggestion
```